UKHomeOffice/cerberus-service
# Cop 9550 Task list UI as prototype

## Description
This PR updates the task list page UI to **closer** match the one found on the prototype (https://cerberus.prototype.cop.homeoffice.gov.uk/blue/pinned/task-list)
- JIRA Ticker: https://support.cop.homeoffice.gov.uk/browse/COP-9550

## To Test
- Pull & run against dev.
- Compare the new task list page UI against the current UI.


> **New UI - COP-9550**
![Cerberus UI - COP-9550 - UI as Prototype](https://user-images.githubusercontent.com/19333750/145706068-68e62a96-8396-474a-9a34-8274afe161fa.png)


vs.


 > **Current UI**
![Cerberus UI - Current state](https://user-images.githubusercontent.com/19333750/145706080-da701673-14b6-443d-8d3b-10ecdaeb953c.png)



> **Please Note:**
> This ticket performs a UI update to closer match that of the prototype. Do note that this UI update omits the following listed below which may be raised in a new ticket at a later date should the need arises.
> - The select All checkboxes.
> - The checkbox next to each task.
> - The icon next to each task
> - The Ship/ Car/ Van icons.


## Developer Checklist

\* Required
